packages/usa-range/src/index.js

```diff
@@ -0,0 +1,52 @@
+const selectOrMatches = require("../../uswds-core/src/js/utils/select-or-matches");
+const behavior = require("../../uswds-core/src/js/utils/behavior");
+
+const { prefix: PREFIX } = require("../../uswds-core/src/js/config");
+
+const RANGE_CLASSNAME = `${PREFIX}-range`;
+const RANGE = `.${RANGE_CLASSNAME}`;
+
+/**
+ * Called when range slider has optional unit data attribute.
```

Comment:
It would be helpful to be a bit more specific here.
```suggestion
 * Called when range slider includes the data-optional-unit attribute.
```

Reply:
Resolved in [9e0d349](https://github.com/uswds/uswds/pull/5472/commits/9e0d349c6e591069db7a4e237b3ffdb9c19d2257)